lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -442,6 +442,41 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value rhs = convertScalarToDtype(b, loc, payloadArgs[1], dtype);
     return b.create<arith::DivFOp>(loc, lhs, rhs);
   }
+  if (auto divTensorMode = dyn_cast<AtenDivTensorModeOp>(op)) {
+    AtenDivTensorModeOp::Adaptor adaptor(operands);
+    Type dtype = converter->convertType(divTensorMode.getType())
+                     .cast<RankedTensorType>()
+                     .getElementType();
+    if (!dtype.isa<mlir::FloatType>())
+      divTensorMode.emitError("unimplemented: non-floating point dtype");
```

Comment:
`emitError` does not exit out of the function, so every `emitError` must be followed by a `return nullptr` in this function. I noticed that `AtenDivTensorOp` above also does not return early. Can you fix that one too?